### PR TITLE
Contact-form and its validation

### PR DESCRIPTION
**Pull request by samiullah12345678**
In this session, a contact form is added

- the contact form has the required field setup 
- the form has the validation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,7 +180,7 @@ <h2 class="card-header">About me</h2>
       <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </h2>
     </header>
-    <form id="contact-form" action="" target="_self" method="post">
+    <form id="contact-form" action="https://formspree.io/f/meqwrjep" target="_self" method="post">
       <fieldset>
           <input id="forminput1" type="text" name="full_name" placeholder="Full Name" required maxlength="30" />
           <label for="forminput1" class="label-remove">Full Name</label>
```
Comment (osamaashraf6) [inline index.html:186]: - [ ] While you make a great effort 😃 I see that the margin between the submit button and the input is greater than the Figma design .so please try to decrease the margin between the input and the submit button to make it like the Figma Design 😊💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  image    |    image |

Comment (osamaashraf6) [inline index.html:186]: - [ ] Kindly 😃 try to stick to the Figma Design at the styling as you see here the padding of the section contact is greater than the Figma, so please try to decrease the padding in this section to make it like the Figma Design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  image  |   image  |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @samiullah12345678 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️


- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow
- [x] Nice design.




## Required Changes ♻️

- Check the comments under the review😊⤵️

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (samiullah12345678) [reply index.html:186]: sir this version is only optimized for the mobile

Comment (samiullah12345678) [reply index.html:186]: done sir ans for padding it is only the mobile vrsion

**Review by didierganthier — APPROVED** [review]
Hi @samiullah12345678 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**